misk-jobqueue-testing/src/main/kotlin/misk/jobqueue/FakeTransactionalJobQueue.kt

```diff
@@ -73,15 +75,16 @@ class FakeTransactionalJobQueue @Inject constructor(
   /** Returns all jobs that were handled. */
   fun handleJobs(
     queueName: QueueName,
-    assertAcknowledged: Boolean = true
+    assertAcknowledged: Boolean = true,
+    retries: Int = 1
   ): List<FakeJob> {
     val jobHandler = jobHandlers.get()[queueName]!!
     val jobs = jobQueues[queueName] ?: return listOf()
 
     val result = mutableListOf<FakeJob>()
     while (true) {
       val job = jobs.poll() ?: break
-      jobHandler.handleJob(job)
+      retry(retries,  FlatBackoff(Duration.ofMillis(100))) { jobHandler.handleJob(job) }
```

Comment:
Just curious, where did the constant `100` come from? Is that what is used elsewhere?

Reply:
Pretty arbitrary, but small timeouts like this are used in the other tests.